### ISSUE TITLE
Events mysteriously modified by SYSTEM user

### ISSUE THREAD

**Issue by RichieB2B** [MISP/MISP#3926]
Some events get mysterious updates where nothing is actually changed except the timestamp.
An example is event [5afaa510-45b8-4ed5-8199-78e5ac100a5a](https://misp.circl.lu/events/view/11379), its Event History on our server looks like:
![misp-event-logs](https://user-images.githubusercontent.com/1461970/49651082-89138e80-fa2e-11e8-8b60-9bd89f0b449f.png)

In this case on the CIRCL server it seems that the event is alternating "modified" by several organizations. Could this be a server sync issue?
![misp-event-logs2](https://user-images.githubusercontent.com/1461970/49651351-31295780-fa2f-11e8-9875-6dc28b5e9ec4.png)

**Comment by iglocska** [comment]
Yeah need to look into this. It looks like something triggers the events' to receive updated publish_timestamps on a push/pull. It's pretty benign apart from the logs being filled with junk.
